### ISSUE TITLE
Cannot install flynn because  file not found: /v20170116.0p1/flynn-host.gz

### ISSUE THREAD

**Issue by regedarek** [flynn/flynn#3865]
During fresh install flynn on ubuntu server 16.04
```
Detecting authentication
Verified authentication for flynn@164.132.192.16
Allocating domain
Configuring DNS
Installing flynn
Installing flynn on 164.132.192.16
Running `curl -fsSL -o /tmp/install-flynn https://dl.flynn.io/install-flynn && sudo bash /tmp/install-flynn --clean` on 164.132.192.16
<SUDO_PROMPT>
[1;33m===> 22:10:26.226 *** WARNING ***[0m
[1;33m===> 22:10:26.228 About to stop Flynn and remove all existing data[0m
[1;33m===> 22:10:26.229 Are you sure this is what you want?[0m
(yes/no): [1;32m===> 22:10:28.409 stopping flynn-host daemon[0m
Failed to stop flynn-host.service: Unit flynn-host.service not loaded.
Failed to execute operation: No such file or directory
[1;32m===> 22:10:28.423 killing old containers[0m
No .containerinit found running; none killed.
[1;32m===> 22:10:28.445 destroying ZFS volumes[0m
[1;32m===> 22:10:28.452 running "zpool destroy flynn-default"[0m
[1;32m===> 22:10:28.495 removing Flynn files and directories[0m
[1;32m===> 22:10:28.496 running "rm -rf /usr/local/bin/flynn* /var/lib/flynn /etc/flynn /etc/init/flynn-host.conf /lib/systemd/system/flynn-host.service"[0m
[1;32m===> 22:10:28.503 Flynn successfully removed[0m
[1;32m===> 22:10:28.505 installing ZFS[0m
[1;32m===> 22:10:28.507 running "apt-get update"[0m
Stary:1 http://nova.clouds.archive.ubuntu.com/ubuntu xenial InRelease
Pobieranie:2 http://nova.clouds.archive.ubuntu.com/ubuntu xenial-updates InRelease [102 kB]
Stary:3 http://nova.clouds.archive.ubuntu.com/ubuntu xenial-backports InRelease
Pobieranie:4 http://security.ubuntu.com/ubuntu xenial-security InRelease [102 kB]
Pobrano 204 kB w 0s (385 kB/s)
Czytanie list pakietów...
[1;32m===> 22:10:30.330 installing runtime dependencies[0m
[1;32m===> 22:10:30.331 running "apt-get install --yes iptables zfsutils-linux ntp"[0m
Czytanie list pakietów...
Budowanie drzewa zależności...
Odczyt informacji o stanie...
iptables is already the newest version (1.6.0-2ubuntu3).
ntp is already the newest version (1:4.2.8p4+dfsg-3ubuntu5.3).
zfsutils-linux is already the newest version (0.6.5.6-0ubuntu15).
0 aktualizowanych, 0 nowo instalowanych, 0 usuwanych i 102 nieaktualizowanych.
[1;32m===> 22:10:30.716 loading zfs kernel module[0m
[1;32m===> 22:10:30.717 running "modprobe zfs"[0m
[1;32m===> 22:10:30.719 downloading flynn-host binary to tmp dir[0m
[1;32m===> 22:10:31.448 verifying flynn-host binary checksum[0m
[1;32m===> 22:10:31.479 running "gunzip flynn-host.gz"[0m
[1;32m===> 22:10:31.647 running "chmod +x flynn-host"[0m
[1;32m===> 22:10:31.649 setting release update channel to "stable"[0m
[1;32m===> 22:10:31.653 downloading Flynn components[0m
[1;32m===> 22:10:31.658 running "./flynn-host download --repository https://dl.flynn.io/tuf --tuf-db /etc/flynn/tuf.db --config-dir /etc/flynn --bin-dir /usr/local/bin"[0m
t=2017-01-23T22:10:31+0100 lvl=info msg="initializing ZFS volumes"
t=2017-01-23T22:10:31+0100 lvl=info msg="determining ZFS zpool size"
t=2017-01-23T22:10:31+0100 lvl=info msg="using ZFS zpool size 7238581862"
t=2017-01-23T22:10:31+0100 lvl=dbug msg="zfs get all -Hp flynn-default" package=zfs
t=2017-01-23T22:10:31+0100 lvl=dbug msg="zpool create flynn-default -mnone /var/lib/flynn/volumes/zfs/vdev/flynn-default-zpool.vdev" package=zfs
t=2017-01-23T22:10:31+0100 lvl=dbug msg="zfs get all -Hp flynn-default" package=zfs
t=2017-01-23T22:10:31+0100 lvl=dbug msg="zfs get all -Hp flynn-default/data" package=zfs
t=2017-01-23T22:10:31+0100 lvl=dbug msg="zfs create flynn-default/data" package=zfs
t=2017-01-23T22:10:31+0100 lvl=dbug msg="zfs get all -Hp flynn-default/data" package=zfs
t=2017-01-23T22:10:31+0100 lvl=dbug msg="zfs get all -Hp flynn-default/squashfs" package=zfs
t=2017-01-23T22:10:31+0100 lvl=dbug msg="zfs create flynn-default/squashfs" package=zfs
t=2017-01-23T22:10:31+0100 lvl=dbug msg="zfs get all -Hp flynn-default/squashfs" package=zfs
t=2017-01-23T22:10:31+0100 lvl=dbug msg="zfs get all -Hp flynn-default/ext2" package=zfs
t=2017-01-23T22:10:31+0100 lvl=dbug msg="zfs create flynn-default/ext2" package=zfs
t=2017-01-23T22:10:31+0100 lvl=dbug msg="zfs get all -Hp flynn-default/ext2" package=zfs
t=2017-01-23T22:10:31+0100 lvl=info msg="initializing TUF client"
t=2017-01-23T22:10:35+0100 lvl=info msg="getting configured release channel"
t=2017-01-23T22:10:35+0100 lvl=info msg="determining latest version of stable release channel"
t=2017-01-23T22:10:35+0100 lvl=info msg="latest stable version is v20170116.0p1"
t=2017-01-23T22:10:35+0100 lvl=info msg="downloading components with version v20170116.0p1"
t=2017-01-23T22:10:35+0100 lvl=info msg="downloading binaries to /usr/local/bin"
t=2017-01-23T22:10:35+0100 lvl=eror msg="error downloading binaries" err="tuf: file not found: /v20170116.0p1/flynn-host.gz"
22:10:35.286175 host.go:163: tuf: file not found: /v20170116.0p1/flynn-host.gz
[1;31m===> 22:10:35.295 ERROR: failed to run "./flynn-host download --repository https://dl.flynn.io/tuf --tuf-db /etc/flynn/tuf.db --config-dir /etc/flynn --bin-dir /usr/local/bin", exit status 1[0m
Installing flynn on 164.132.192.16
Running `curl -fsSL -o /tmp/install-flynn https://dl.flynn.io/install-flynn && sudo bash /tmp/install-flynn --clean` on 164.132.192.16
<SUDO_PROMPT>
Using cached password
[1;33m===> 22:10:45.739 *** WARNING ***[0m
```

**Comment by lmars** [comment]
@regedarek sorry about this, we have had some issues with the release system today, I am investigating.

**Comment by lmars** [comment]
This is now fixed with the latest `v20170116.0p2` release.
